scripts/lib.star

```diff
@@ -842,13 +842,10 @@ def get_windows_steps(edition, version_mode, is_downstream=False):
             'rm gcpkey.json',
             'cp C:\\App\\nssm-2.24.zip .',
             '.\\grabpl.exe windows-installer --edition {} --build-id $$env:{}'.format(edition, build_no),
+            '$$fname = ((Get-Childitem grafana*.msi -name) -split "`n")[0]',
+            'gsutil cp $$fname gs://grafana-downloads/{}/{}/'.format(edition, version_mode),
+            'gsutil cp "$$fname.sha256" gs://grafana-downloads/{}/{}/'.format(edition, version_mode),
```

Comment:
out of curiosity: why only in the 2nd command the source file should be quoted?

Reply:
@papagian Good question - it's necessary in this case, to stop PowerShell from interpreting it as getting property `sha256` of `$fname`. Make sense?

Reply:
ah you mean for making the concatenation working; it makes sense